web/src/components/CategoryEntry.tsx

```diff
@@ -117,6 +117,15 @@ const CategoryEntry = ({ category, contentWithoutHtml, subCategories, query }: P
       textToHighlight={textToHighlight}
       highlightStyle={{ backgroundColor: theme.colors.backgroundColor, fontWeight: 'bold' }}
     />
+  ) : (
+    query && (
+      <p>
+        {new ContentMatcher()
+          .getWords(parseHTML(category.content).slice(0))
```

Comment:
Why do we need the `.slice(0)`?
~~Also isn't this the same as directly using `contentWithoutHtml`?~~
Edit: I see, we only pass `contentWithoutHtml` if it is a match by content. Imo we should make it uniform and either pass `contentWithoutHtml` for all categories or just parse it for every category here. Since we only need `contentWithoutHtml` for the search and not the regular category list I'd prefer the former.

Reply:
Done